python/cugraph/cugraph/gnn/dgl_extensions/feature_storage.py

```diff
@@ -67,7 +66,7 @@ def fetch(self, indices, device=None, pin_memory=False, **kwargs):
         # Default implementation uses synchronous fetch.
 
         indices = cp.asarray(indices)
-        if isinstance(self.pg, MGPropertyGraph):
+        if type(self.pg).__name__ in "MGPropertyGraph":
```

Comment:
I think this is a potential subtle bug, or did you mean to use `in`?  Here's an example:
```python
>>> x=2
>>> type(x).__name__ in "int"
True
>>> type(x).__name__ in "iiinttt"
True
>>> type(x).__name__ == "int"
True
>>> type(x).__name__ == "iiinttt"
False
```

Reply:
A more relevant example :)
```python
>>> pg=PropertyGraph()
>>> type(pg).__name__ in "MGPropertyGraph"
True
```

Reply:
Resolved